setup.py

```diff
@@ -33,7 +33,7 @@
         'scikit-image',
         'scikit-learn',
         'scipy',
-        'shapely',
+        'shapely'
```

Comment:
Keep the trailing comma, it's [pep8 recommended](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas) so you do not need to change this line when you append another item to the list.